doc/tutorials/dune-package-management/concepts.md

```diff
@@ -0,0 +1,40 @@
+# Concepts
+
+## Motivation
```

Comment:
I don't think we need to motivate package management, but we could have a section that explains the history of this in the ecosystem.

Newcomers and most people that want to get started with it don't _need_ to know what opam is, just how to use package management with dune.

Reply:
I think describing why we're doing this and why this is an improvement is important, given this tutorial is not only meant for newcomers but also for existing users of OCaml that use OPAM.

Reply:
Maybe the comparison to opam should be a separate section?

Reply:
Yes, sleeping over it and reading the Diátaxis document I think the right thing is to remove it from the tutorial. There is probably a point to describe the how package management works in the documentation somewhere else but it is not in the tutorial.